### DISCUSSION TITLE
docker installs ngrinder3.4, character

### DISCUSSION THREAD

**Discussion by zhougy1989** [naver/ngrinder#808] [Q&A]
When docker installs ngrinder 3.4, the log in the version console and log file will be displayed in Chinese??, Is there a solution?
![image](https://user-images.githubusercontent.com/50607317/121354236-43b6db80-c961-11eb-8fbf-51c0e6bdb879.png)
@junoyoon @donggyu04 
thks

**Comment by donggyu04** [comment]
@zhougy1989 

I have no problem encoding chinese. please use the latest version of the image. (I have been used 3.5.5-p1)
and check [similar issue](https://github.com/naver/ngrinder/issues/445).

![image](https://user-images.githubusercontent.com/14273601/121367693-7bc81980-c975-11eb-9da0-f86493cb8c50.png)
